streaming/src/main/scala/org/apache/gearpump/streaming/dsl/window/impl/WindowRunner.scala

```diff
@@ -73,6 +73,25 @@ case class AndThen[IN, MIDDLE, OUT](left: WindowRunner[IN, MIDDLE],
   }
 }
 
+class DirectWindowRunner[IN, OUT](fnRunner: FunctionRunner[IN, OUT])
```

Comment:
@huafengw I'm a bit stuck for the name here. This is actually not a window runner. Any suggestions ?

Reply:
Maybe change `WindowRunner` to a more generic name? Then `DirectWindowRunner` can be a non-window runner.